iothub/service/src/Messaging/Models/MessageConverter.cs

```diff
@@ -113,7 +113,7 @@ public static void UpdateMessageHeaderAndProperties(AmqpMessage amqpMessage, Mes
         }
 
         /// <summary>
-        /// Copies the Message instance's properties to the AmqpMessage instance.
+        /// Copies the message instance's properties to the amqp message instance.
```

Comment:
Acronyms all upper case when used in text (not variable names).

```suggestion
        /// Copies the message instance's properties to the AMQP message instance.
```